system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -171,6 +171,34 @@ def test_file_based_external_account(
             },
         )
 
+# This test makes sure that setting a token lifetime works
+# for service account impersonation.
+def test_file_based_external_account_with_configure_token_lifetime
```

Comment:
This is missing a `(` and causing the system tests to fail

Reply:
Fixed